Libraries/LogBox/LogBox.js

```diff
@@ -12,8 +12,6 @@
 
 import Platform from '../Utilities/Platform';
 import RCTLog from '../Utilities/RCTLog';
-import * as LogBoxData from './Data/LogBoxData';
```

Comment:
This should be handled by the inline requires feature. We don't like manually inlining requires if we can avoid it.

Reply:
Sadly this is not enabled by default in OSS so I think doing it manually in this case is fine since it is not only for perf improvements.